### ISSUE TITLE
[Training Issue] loss can't converge and not stable

### ISSUE THREAD

**Issue by xi3299** [eric612/MobileNet-YOLO#122]
Model : yolov3-lite

Backbone : mobilenet

Questions :
Hello, 
I want to train mobilenet-yolov3_lite for person class of VOC. The batch_size has been set to 6*6=36, after have trained 17000 iterations, the loss is always net stable and too big (50 to 1000). But also, the caffemodel can't detect things. I am now wondering how to set correctly the parameters in solver.prototxt file.
There are 1,677 images in test dataset and  11,711 images in training dataset.
Here, I put the solver.prototxt and deploy.prototxt. 

[mobilenet_yolov3_lite_fn_solver.txt](https://github.com/eric612/MobileNet-YOLO/files/3251757/mobilenet_yolov3_lite_fn_solver.txt)
[mobilenet_yolov3_lite_bn_deploy.txt](https://github.com/eric612/MobileNet-YOLO/files/3251762/mobilenet_yolov3_lite_bn_deploy.txt)
[mobilenet_yolov3_lite_train.txt](https://github.com/eric612/MobileNet-YOLO/files/3251763/mobilenet_yolov3_lite_train.txt)
[mobilenet_yolov3_lite_test.txt](https://github.com/eric612/MobileNet-YOLO/files/3251764/mobilenet_yolov3_lite_test.txt)

Thanks a lot for any solution!

**Comment by eric612** [comment]
* The major problem is the anchor scale , you need re-calculation the k-mean anchors , follow this issue 
https://github.com/eric612/MobileNet-YOLO/issues/92

* The minor problem was I changed the jitter code , so you may need change the jitter scale to 0.7 and expand ratio to 4.0 if you use new version and train 20 classes on VOC

**Comment by xi3299** [comment]
Thanks a lot for your response!  I has recalculated the anchors. ^^
I don't know my VOC version is new or not, but I just train one class on VOC. So should I still change jitter scale and ration to 4.0?

**Comment by xi3299** [comment]
>     * The major problem is the anchor scale , you need re-calculation the k-mean anchors , follow this issue
>       #92
> 
>     * The minor problem was I changed the jitter code , so you may need change the jitter scale to 0.7 and expand ratio to 4.0 if you use new version and train 20 classes on VOC

I modified according to all your suggestions. And now it works good. Thank you so much!^^
The loss is still a bit big now, do you have some ideas about finetuning?
